OIPA/api/transaction/serializers.py

```diff
@@ -229,7 +231,7 @@ def validate(self, data):
             # data.get('sector', {}).get('code'),
             # data.get('sector', {}).get('vocabulary'),
             # data.get('sector', {}).get('vocabulary_uri'),
-            data.get('recipient_country', {}).get('code'),
```

Comment:
recipient_countries containing all the recipient countries related to te transaction.
recipient_country is setting one countrry to the transaction.

unit test added